src/PowerShellEditorServices.VSCode/CustomViews/CustomViewMessages.cs

```diff
@@ -14,13 +12,6 @@ namespace Microsoft.PowerShell.EditorServices.VSCode.CustomViews
     /// </summary>
     public class NewCustomViewRequest
     {
```

Comment:
Maybe these should still hold a constant for the request string:

```csharp
internal const string RequestId = "powerShell/newCustomView";

public static string Id => RequestId;
```

(internal constant for our use, readonly static property for public use so folks don't have to recompile if it's changed)

Reply:
so Id is already defined so I went with method.

Is the internal constant really necessary? over just:

```
public static string Method => "foo"
```

Reply:
Nah you're right, probably not, it'll almost definitely be inlined.